app/templates/calendar_day_view.html

```diff
@@ -41,7 +41,10 @@
     </div>
     <div class="event-grid">
       {% for event, attr in events %}
-      <div id="event{{event.id}}" class="d-flex flex-column text-truncate px-2 event"
+          {% if event.deleted_date != None %}
+                {% set deleted_event = 'deleted_event' %}
```

Comment:
`{% set deleted_event = 'deleted_event' if event.deleted_date != None else '' %}`
(I think the syntax is : ? for jijna)

Reply:
Thanks